packages/grid/src/BaseCell.css

```diff
@@ -0,0 +1,59 @@
+.uitkGridBaseCell {
+  box-sizing: border-box;
+  overflow: hidden;
+  margin: 0;
+  padding: 0;
+
+  border-style: solid;
+  border-top-color: var(--row-border-color);
+  border-right-color: var(--column-separator-color);
+  border-width: 1px 1px 0 0;
+
+  white-space: nowrap;
+  position: relative;
+  font-size: var(--uitkGrid-fontSize);
+  font-weight: normal;
+  z-index: var(--uitkGrid-cell-zIndex);
+}
+
+.uitkGridBaseCell-selectedCell {
+  background-color: var(--uitkGrid-cell-backgroundColor-selected);
```

Comment:
have these as background instead of background-color (and same for token) - gives flexibility. is there a default (private token) here or is there only a color if defined? (since the --uitkGrid token is the API)

Reply:
i think possibly you meant here for it to be --grid (looking at the others below) which is the token we define, --uitkGrid is for consumers. i put this in this issue #377 previously, but could be fixed in this PR